src/main/java/org/myrobotlab/service/meta/abstracts/MetaData.java

```diff
@@ -8,200 +8,137 @@
 import java.util.Set;
 import java.util.TreeMap;
 
-import org.myrobotlab.codec.CodecUtils;
 import org.myrobotlab.framework.ServiceReservation;
 import org.myrobotlab.framework.repo.ServiceArtifact;
-import org.myrobotlab.framework.repo.ServiceData;
 import org.myrobotlab.framework.repo.ServiceDependency;
 import org.myrobotlab.framework.repo.ServiceExclude;
 import org.myrobotlab.logging.LoggerFactory;
 import org.slf4j.Logger;
 
 /**
- * MetaData describes most of the data about a service that is static.
- * It's dependencies, categories and other meta information.
+ * MetaData describes most of the data about a service that is static. It's
+ * dependencies, categories and other meta information.
  * 
- * It also describes its list of peers. They define a list of group this 
- * service expects to interact with.  When a service is started it gets
- * a copy of MetaData based on its instance name. When a new instance is
- * created a list of ServiceData.overrides are consulted, to allow the user
- * to override actual name and type information.
+ * It also describes its list of peers. They define a list of group this service
+ * expects to interact with. When a service is started it gets a copy of
+ * MetaData based on its instance name. When a new instance is created a list of
+ * ServiceData.overrides are consulted, to allow the user to override actual
+ * name and type information.
  * 
  */
-public class AbstractMetaData implements Serializable {
+public class MetaData implements Serializable {

```

Comment:
MetaData is such an overused term. we should probably rename this to ServiceMetaData ... calling it just "metadata" is very vague to a noob, would think.